lua/entities/wtib_harvester_medium/init.lua

```diff
@@ -64,12 +63,19 @@ function ENT:Harvest()
 
 					WTib.ConsumeResource(self,"energy",Drain*1.2)
 					WTib.SupplyResource(self,"RawTiberium",Drain)
-					v:SetTiberiumAmount(v:GetTiberiumAmount()-Drain)
+					v:DrainTiberiumAmount(Drain)
+					
+					local ed = EffectData()
+						ed:SetEntity(self)
+						ed:SetStart(self:GetPos()+(self:GetUp()*self.Focus))
+						ed:SetOrigin(v:GetPos())
+						ed:SetScale(2)
+						ed:SetMagnitude(1)
+					util.Effect("wtib_debugtrace", ed)
```

Comment:
I used to have a harvest effect, which was green particles moving from the crystal to the harvester, but for some reason I removed that. I guess for now this will do.